src/openzaak/conf/includes/base.py

```diff
@@ -314,6 +314,7 @@
 ]
 
 SESSION_COOKIE_NAME = "openzaak_sessionid"
+SESSION_ENGINE = "django.contrib.sessions.backends.cache"
```

Comment:
Is this needed?

Reply:
@stevenbal you prolly discussed offline but why was this needed?